docs/phdata.rst

```diff
@@ -55,24 +55,24 @@ This section describes the layout and fields in the **/photon_data** group.
 
 Mandatory fields:
 
-- **timestamps**: (array) the photon timestamps.
+- **timestamps**: (array) photon timestamps.
 - **timestamps_specs/**
-    - **timestamps_unit**: (float) timestamps units in *seconds*.
+    - **timestamps_unit**: (float) timestamp units in *seconds*.
 
 Optional if there is only 1 detector, otherwise mandatory:
 
-- **detectors**: (array of integers) the detector ID for each timestamp.
+- **detectors**: (array of integers) detector ID for each timestamp.
 
-When the dataset contains nanotime information, the following
-fields must be present:
+When the dataset contains nanotime information (i.e. arrival time of each photon with respect to a laser pulse), the following
+fields must be present (**nanotime can be -in principle- counted from the previous or next laser pulse: how is this defined in the format?**)
 
```

Comment:
Good point, I can add a boolean specs `nanotimes_specs/time_reversed` that stated whether the time axis should be reversed.